src/components/RefinementList/RefinementList.js

```diff
@@ -30,9 +30,11 @@ class RefinementList extends Component {
     let subItems;
     const hasChildren = facetValue.data && facetValue.data.length > 0;
     if (hasChildren) {
+      const cssClasses = { ...this.props.cssClasses, root: '' };
```

Comment:
Removing the `root` property is more suited here because it can be type-checked.

```ts
const { root, ...cssClasses } = this.props.cssClasses;
```

Reply:
```suggestion
      const { root, ...cssClasses } = this.props.cssClasses;
```

suggestion to keep it simple, I agree by the way